tests/Tests/QueryDsl/FullText/Match/MatchUsageTests.cs

```diff
@@ -69,5 +74,16 @@ public class MatchUsageTests : QueryDslUsageTestsBase
 				.Name("named_query")
 				.AutoGenerateSynonymsPhraseQuery(false)
 			);
+
+		//hide
+		[U] public void DeserializeShortForm()
+		{
+			var json = JsonConvert.SerializeObject(new { description = "project description" });
```

Comment:
Move to base and reuse for other DeserializeShortForm tests

Reply:
Addressed in 5445bc9dc1ba2bbc093f43e3c982ebaaecf866ac